src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -234,7 +247,7 @@ private static IEnumerable<IGraphType> GetSchemaTypes(ISchema schema, IServicePr
         /// Returns a dictionary that relates type names to graph types.
         /// </summary>
         protected internal virtual Dictionary<string, IGraphType> Dictionary { get; } = new Dictionary<string, IGraphType>();
-        private readonly Dictionary<Type, IGraphType> _typeDictionary;
```

Comment:
I don't like that fields lost `readonly` modifier.

Reply:
Agreed, but there's nothing I can do about it.